src/integ_test/java/games/strategy/engine/framework/startup/login/ClientLoginIntegrationTest.java

```diff
@@ -114,26 +111,24 @@ public void login_ShouldFailUsingMd5CryptAuthenticatorWhenPasswordDoesNotMatch()
       }
     };
 
-    catchException(() -> newClientMessenger(connectionLogin).shutDown());
-
-    assertThat(caughtException(), is(instanceOf(CouldNotLogInException.class)));
+    assertThrows(CouldNotLogInException.class, () -> newClientMessenger(connectionLogin).shutDown());
   }
 
   @Test
   public void login_ShouldSucceedUsingHmacSha512AuthenticatorWhenPasswordMatches() {
     final IConnectionLogin connectionLogin = new TestConnectionLogin();
 
-    catchException(() -> newClientMessenger(connectionLogin).shutDown());
-
-    assertThat(caughtException(), is(nullValue()));
+    try {
+      newClientMessenger(connectionLogin).shutDown();
+    } catch (final Exception e) {
+      throw new AssertionError(e);
+    }
```

Comment:
If we notice this pattern again in the future, we should consider extracting an `assertNotThrows(Executable)` helper method to avoid the boilerplate.

Reply:
We could also just drop the try block entirely, the only difference being that in tools that differentiate between AssertionErrors and other errors will report exceptions as errors instead of failures.
On the other hand: A failing test is a failing test no matter what.

Reply:
True, we could.  You already pointed out the fact that the graphical JUnit runner will show the test as errored rather than failed.  However, I think a more compelling reason to not just let the exception escape the method is that a test without assertions becomes hard to read later on.  Someone may be left wondering what exactly is being tested.  So we should always be asserting _something_.